src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -1275,6 +1275,10 @@ HeapWord* ShenandoahHeap::allocate_new_gclab(size_t min_size,
 HeapWord* ShenandoahHeap::allocate_new_plab(size_t min_size,
                                             size_t word_size,
                                             size_t* actual_size) {
+  size_t words_in_card = CardTable::card_size_in_words();
```

Comment:
Add a comment such as:

```
// Align requested sizes to card sized multiples
```

(Which brings up the question whether the alignment might profitably be made in the caller -- not suggesting this, just wondering; doesn't need to be done here or now.)